tests/scripts/thread-cert/Cert_5_3_01_LinkLocal.py

```diff
@@ -75,6 +99,105 @@ def test(self):
         # 8
         self.assertTrue(self.nodes[LEADER].ping(config.LINK_LOCAL_All_THREAD_NODES_MULTICAST_ADDRESS))
```

Comment:
use `LINK_LOCAL_ALL_NODES_ADDRESS` and `LINK_LOCAL_ALL_ROUTERS_ADDRESS` for above `ff02::1` and `ff02::2` respectively